make-rules/prep.mk

```diff
@@ -25,11 +25,16 @@
 
 include $(WS_MAKE_RULES)/prep-download.mk
 include $(WS_MAKE_RULES)/prep-hg.mk
+include $(WS_MAKE_RULES)/prep-git.mk
 include $(WS_MAKE_RULES)/prep-svn.mk
 include $(WS_MAKE_RULES)/prep-unpack.mk
 include $(WS_MAKE_RULES)/prep-patch.mk
 
+$(eval $(call eval-download-rules))
```

Comment:
Why do we need these? It's already called in prep-download.mk.
Perhaps, missing some solaris-userland commit?

Reply:
Okay. Then I think I understand...
BUT then there should be a change in prep-hg-rules.mk calling the respective function there directly as it is done in prep-download.mk, or not?